packages/@nucleus/tests/dummy/app/templates/components/theme-toggle.hbs

```diff
@@ -0,0 +1,7 @@
+<button class="mode-toggle" onclick={{action "onToggle"}}>
+  {{#if dark}}
+   {{nucleus-icon name="light" customClass="toggle-icon" size="large"}}
```

Comment:
`{{nucleus-icon name="{{if dark "light" "dark"}}" customClass="toggle-icon" size="large"}}` ?

Reply:
Can this be done? Handlebars within handlebars? Wouldn't it throw an error?

Reply:
Sorry, CP instead?

Reply:
Sorry, my bad. Got it confused with angle bracket syntax. Use CP instead may be?